lib/std/zig/tokenizer.zig

```diff
@@ -11,6 +11,7 @@ pub const Token = struct {
     };
 
     pub const keywords = std.ComptimeStringMap(Tag, .{
+        .{ "addrspace", .keyword_addrspace },
```

Comment:
consistency nit: example `\\const y: []allowzero align(8) addrspace(.generic) const volatile u32 = undefined;`
shows align before addrspace.

Reply:
This remainder of this list sorted alphabetically, in which case addrspace should be placed before align.